### PR TITLE
clean up the code and variable bug fix

### PR DESCRIPTION
**Pull request by DMGarcia17**

### CODE REVIEW

File: app/controllers/api/authController.js [hunk 1]
```diff
@@ -6,14 +6,23 @@ let asyncHandler = require('../../utils/asyncHandler');
 router.post(
     '/login',
     asyncHandler(async (req, res) => {
-        const result = await authService.login(req, res);
-        res.send({success:result.success, message:result.message, token: result.token});
+        console.log(req.checkBody('txtPassword'));
```
Comment (edgarMejia) [inline app/controllers/api/authController.js]: Please remove this console.log

File: app/services/userService.js [hunk 2]
```diff
@@ -16,10 +16,9 @@ class userService {
         }
 
         user_app.password = await bcrypt.hash(plainPassword, CONSTANTS.saltRounds); //.then(hash => {
-        let result = await db.pg.user_app.save(user_app);
-        
-        return result;
+        console.log(user_app);
```
Comment (edgarMejia) [inline app/services/userService.js]: Remove this console.log too

File: app/utils/constants.js [hunk 3]
```diff
@@ -4,9 +4,9 @@ let domain = process.env.APIPERSONA_DOMAIN || 'http://localhost:3000/';
 let postgreHost = process.env.postgre_HOST || 'localhost';
 let postgrePort = process.env.postgre_PORT || '5432';
 
-let postgreDb = process.env.postgreDb || 'angular_crud';
-let postgreUser = process.env.postgreUser || 'edgarmejia';
-let postgrePass = process.env.postgrePass || '123123123';
+let postgreDb = process.env.postgreDb || 'node_api';
+let postgreUser = process.env.postgreUser || 'postgres';
+let postgrePass = process.env.postgrePass || '03012019';
```
Comment (edgarMejia) [inline app/utils/constants.js]: Don't change the credentials

File: app/controllers/api/authController.js (hunk 1 above)
Comment (DMGarcia17) [reply app/controllers/api/authController.js]: done

File: app/services/userService.js (hunk 2 above)
Comment (DMGarcia17) [reply app/services/userService.js]: done

File: app/utils/constants.js (hunk 3 above)
Comment (DMGarcia17) [reply app/utils/constants.js]: done

**Pull request merged**